notebooks/4_2_combinatorial_library_properties.ipynb

```diff
@@ -34,24 +34,31 @@
   },
```

Comment:
<p>Can you run this cell again so it doesn't display this message? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/KinFragLib/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>